src/pyb2b/xml/ForecastFlightPlanRequest.xml

```diff
@@ -0,0 +1,31 @@
+ <flig:FlightListByAerodromeRequest xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
+ xmlns:flig="eurocontrol/cfmu/b2b/FlightServices">
+    <!--Optional:-->
+    <endUserId>bcramet</endUserId>
```

Comment:
careful!

Reply:
oops!
I fixed and squashed commits to remove it